src/v2/Apps/Partners/Routes/GalleriesRoute.tsx

```diff
@@ -34,6 +34,7 @@ const GalleriesRoute: React.FC<GalleriesRouteProps> = ({ viewer }) => {
       <MetaTags
         title="Galleries | Artsy"
         description="Browse art and design galleries around the globe by location and specialty and explore artists, artworks, shows, and fair booths."
+        pathname="galleries"
```

Comment:
I believe this needs a leading slash `/galleries`

Reply:
oh right!

Reply:
it actually doesnt. it detects and adds if needed